scripts/pre-push

```diff
@@ -3,5 +3,8 @@
 # Can be used as a pre-push hook
 # Just symlink this file to .git/hooks/pre-push
 
-echo "Running style check..."
-./scripts/style-check.sh
+echo "Running linter..."
+luacheck .
+
+echo "Auto formatting code..."
+stylua .
```

Comment:
I think it requires the --check option to abort the commit. Auto Format would be only useful for auto-ammend or auto-commit.

Reply:
oh you are right, I was confused with this being run before a commit